diff --git a/node_modules/react-native-thumbnail-video/src/components/thumbnail.js b/node_modules/react-native-thumbnail-video/src/components/thumbnail.js
index bc53a4f..7b04e09 100644
--- a/node_modules/react-native-thumbnail-video/src/components/thumbnail.js
+++ b/node_modules/react-native-thumbnail-video/src/components/thumbnail.js
@@ -4,14 +4,15 @@ import {
   TouchableOpacity,
   ImageBackground,
   Image,
-  ViewPropTypes,
-  ImagePropTypes,
+  // ViewPropTypes,
+  // ImagePropTypes,
   Linking,
   StyleSheet,
 } from 'react-native';
 
 import { DEFAULT_WIDTH, TYPES } from './constants';
 import { getVideoId } from '../helpers';
+import { ViewPropTypes, ImagePropTypes } from 'deprecated-react-native-prop-types';
 
 export default class Thumbnail extends PureComponent {
   constructor(props) {
@@ -34,7 +35,8 @@ export default class Thumbnail extends PureComponent {
       PropTypes.number,
       PropTypes.string,
     ]),
-    iconStyle: Image.propTypes.style,
+    // iconStyle: Image.propTypes.style,
+    iconStyle: ImagePropTypes.style,
     onPress: PropTypes.func,
     onPressError: PropTypes.func,
     style: ViewPropTypes.style,
